testing/SPEC-CONFORMANCE.md

```diff
@@ -6,52 +6,51 @@ The conformance requirements for EBU-TT Part 3 derive from the specification its
+|R33|3.2.2.1|The `ttp:timeBase` element is as defined in [EBUTT1] with the addition that all time expressions of `dur` attributes shall denote a relative coordinate on the same timeline as the `begin` and `end` attributes.|`bdd/features/timing/resolved_times.feature`|
 |R34|3.2.2.1|`ebuttp:sequenceIdentifier` Cardinality 1..1|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
 |R35|3.2.2.1|The sequence to which every document belongs shall be identified using the `ebuttp:sequenceIdentifier` attribute.|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
 |R36|3.2.2.1|`ebutt:sequenceNumber` Cardinality 1..1|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
 |R37|3.2.2.1|~~Every document with the same `ebuttp:sequenceIdentifier` shall be uniquely numbered using the `ebuttp:sequenceNumber` attribute.~~|Deprecated. See R107|
 |R38|3.2.2.2|The following additional metadata elements are defined by this specification. If used these elements shall be the first children of the `ebuttm:documentMetadata` element in the order documented below, after any metadata elements defined by EBU-TT Part 1 Tech 3350.|`bdd/features/validation/documentMetadata_elements_order.feature` `(In)valid documentMetadata elements order`|
 |R39|3.2.2.2|Order of metadata elements (if present) in `ebuttm:documentMetadata` is: `ebuttm:originalSourceServiceIdentifier`, `ebuttm:intendedDestinationServiceIdentifier`, `ebuttm:documentFacet`, `ebuttm:trace`. |`bdd/features/validation/documentMetadata_elements_order.feature` `(In)valid documentMetadata elements order`|
-|R40|3.2.2.2|`ebuttm:documentFacet` : Each distinctly identified `facet` that is summarised shall have a separate `documentFacet` element.| |
-|R41|3.2.2.2|Documents shall NOT contain more than one `documentFacet` element referring to the same term, where the term is identified by the combination of the element contents and the value of the link attribute.| |
-|R42|3.2.2.2.1|~~ `ebuttm:trace` `action` attribute Cardinality 1..1 ~~ Merged with `ebuttm:appliedProcessing`|`bdd/features/validation/trace.feature` `(In)valid trace attributes`|
+|R40|3.2.2.2|`ebuttm:documentFacet` : Each distinctly identified `facet` that is summarised shall have a separate `documentFacet` element.|NOTE:Semantic validation for Facet planned to R4|
+|R41|3.2.2.2|Documents shall NOT contain more than one `documentFacet` element referring to the same term, where the term is identified by the combination of the element contents and the value of the link attribute.|NOTE: Semantic validation for Facet planned to R4|
+|R42|3.2.2.2.1|~~`ebuttm:trace` `action` attribute Cardinality 1..1~~ Merged with `ebuttm:appliedProcessing`|`bdd/features/validation/trace.feature` `(In)valid trace attributes`|
```

Comment:
I think the BDD feature file name was changed to match, so it is something like `applied-processing.feature` now.